book/src/filters.md

```diff
@@ -323,7 +370,49 @@ Ugly: <script>var data = "{{data|json}}";</script>
 Ugly: <script>var data = '{{data|json|safe}}';</script>
 ```
 
+### `markdown`
+[#markdown]: #markdown
+
+Enabling the `markdown` feature fill enable the use of the `markdown` filter.
+This will render a value using a [Github flavored commonmark](https://docs.rs/comrak/0.12.*/comrak/) syntax.
+By default the extensions “autolink”, “strikethrough”, “tagfilter”, and “table” are enabled.
+Any raw HTML gets escaped.
+
+```jinja
+{{ "**<i>Hello</i>, world!**"|markdown }}
+```
+
+Output:
+
+```
+<p><strong>&lt;i&gt;Hello&lt;/i&gt;, world!</strong></p>
+```
+
+You can change the default settings by supplying [custom options](https://docs.rs/comrak/0.12.*/comrak/struct.ComrakRenderOptions.html),
+e.g. to enable unsafe raw HTML:
+
+```
+let options = &ComrakOptions {
+    render: ComrakRenderOptions {
+        unsafe_: true,
+        ..Default::default()
+    },
+    ..Default::default()
+};
+
+…
+
+{{ "<script>alert('Hello world')</script>"|markdown(options) }}
+```
```

Comment:
Maybe the usage example, should be more copy-pastable. I think it might be confusing, that it's half Rust and half Jinja :)

Reply:
:+1: I added a link to the unit tests. I think this should be the most readable variant, esp. since it should not be too interesting to most users.